post-deployment/openstack/post-deployment.yml

```diff
@@ -0,0 +1,9 @@
+---
+- import_playbook: object-storage.yml
+- import_playbook: infra.yml
+- import_playbook: storage.yml
+- import_playbook: logging.yml
+  when: logging
```

Comment:
Raw variables as conditions throw warnings about being deprecated, and will need `| bool` in future versions of Ansible.

Reply:
Using Ansible 2.9.9 I only get the deprecation warning if I specify logging as `"true"` as opposed to `true` in vars.yml. I will add it regardless in the case that someone specifies logging as a string in vars.yml